bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/AudioWaveUtils.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.audio;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.sound.sampled.AudioFormat.Encoding;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.UnsupportedAudioFileException;
+
+/**
+ * Some utility methods for parsing and cleaning wav files
+ *
+ * @author Gwendal Roulleau - Initial contribution
+ *
+ */
+public class AudioWaveUtils {
```

Comment:
Please add [Null Annotations](https://www.openhab.org/docs/developer/guidelines.html#null-annotations).

```suggestion
@NonNullByDefault
public class AudioWaveUtils {
```

Reply:
Done